src/uShip.Logging/LogBuilders/FluentLogDataBuilder.cs

```diff
@@ -82,6 +84,104 @@ public IFluentLoggerWriter Exception(Exception exception)
                 return this;
             }
 
+            /// <summary>
+            /// Reflectively add non-complex properties (exceptions like DateTime and DateTimeOffset) from an object to the Data dictionary to be included in the logger's message.
+            /// </summary>
+            /// <param name="prefix">Prefix that will be prepended to the property key names in the data dictionary.</param>
+            /// <param name="value">Object/class that we are logging properties from.</param>
+            /// <returns></returns>
+            public IFluentLoggerWriter Data(string prefix, object value)
```

Comment:
Just tested null + string.. in concatenation null behaves the same way as an empty string.